backend/src/routes/account.js

```diff
@@ -98,10 +112,21 @@ const preventLoginForLoggedInUsers = (req, res, next) => {
   next(req.user && new Error('User is already logged in'))
 }
 
+const isEmailVerified = async (req, res, next) => {
+  const user = await User.findOne({ email: req.body.email })
```

Comment:
add `isVerified` to query

Reply:
use `Model.exists()`

we should have a compound index of `email` and `verified` on the User model

Reply:
👌